### PR TITLE
Exclude non-essential files in .gitattributes

### PR DESCRIPTION
**Pull request by vinkla**
[Exclude non-essential files in .gitattributes](https://www.reddit.com/r/PHP/comments/2jzp6k/i_dont_need_your_tests_in_my_production/). This way users of this package doesn't have to download non-essential files such as the `tests` directory or the `.travis.yml` file.

Happy Hacktoberfest!

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/.gitmodules export-ignore
+/.dockerignore export-ignore
+/.travis.yml export-ignore
+/codecov.yml export-ignore
+/Dockerfile export-ignore
+/Performance.php export-ignore
+/php.ini export-ignore
+/phpcs.xml.dist export-ignore
+/phpunit.xml.dist export-ignore
+/README.md export-ignore
```
Comment (felixfbecker) [inline .gitattributes]: Isn't a README always good to have in a distribution?

Comment (vinkla) [reply .gitattributes]: It is common to exclude this file as well. Think of this as when the package is installed in production. When there is no need to check the `vendor` directory.

Comment (felixfbecker) [reply .gitattributes]: True, but does this setting only apply to production environments?

Comment (vinkla) [reply .gitattributes]: Yes, this will apply if the user installs packages with the `--prefer-dist` flag or config.

Comment (felixfbecker) [reply .gitattributes]: gotcha, that makes sense to me.

**Pull request merged**